packages/account-v2/src/containers/PaymentMethods/PaymentMethodForm.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { TPaymentMethod, TPaymentMethodIdentifier, TPaymentMethodInfo } from 'src/types';
+import { PaymentMethodField } from './PaymentMethodField';
+import { PaymentMethodInstructions } from './PaymentMethodInstructions';
+
+type TPaymentMethodFormProps = {
+    paymentMethodDetail: TPaymentMethodInfo;
+};
+
+export const PaymentMethodForm = ({ paymentMethodDetail }: TPaymentMethodFormProps) => {
+    const paymentMethod = paymentMethodDetail.paymentMethod.toLowerCase() as TPaymentMethod;
+
+    return (
+        <div className='flex flex-col gap-24'>
+            <PaymentMethodInstructions paymentMethod={paymentMethod as TPaymentMethod} />
```

Comment:
paymentMethod is already declared of type TPaymentMethod in line 15. do we still need to add typecasting here